tracktotrip/track.py

```diff
@@ -5,7 +5,7 @@
 from os.path import basename
 from datetime import timedelta
 
-import gpxpy
+from . import gpxpy
```

Comment:
`gpxpy` is a dependency, it should not be imported like this.

Reply:
I believe it was introduced by mistake by me. Will fix to the original form.